app/controllers/feedbacks_controller.rb

```diff
@@ -0,0 +1,50 @@
+class FeedbacksController < ApplicationController
+  FEEDBACK_TYPES = %w[freetext satisfaction].freeze
+
+  def create
+    if params[:type].nil? || !FEEDBACK_TYPES.include?(params[:type])
+      raise "Feedback type needs to be specified"
+    end
+
+    @form = params[:type].include?("satisfaction") ? satisfaction_model : freetext_model
+    if @form.valid?
+      if @form.instance_of?(SatisfactionFeedback)
+        SatisfactionFeedback.create!(
+          satisfied: @form.satisfied,
+          level_of_help:,
+          outcome:,
+        )
+      else
+        FreetextFeedback.create!(
+          text: @form.text,
+          page: @form.page,
+          level_of_help:,
+        )
+      end
+    else
+      track_validation_error
+    end
+  end
```

Comment:
If this succeeds, the server should return an appropriate status code. Likewise if it fails. (Practically speaking this means adding something like an explicit `head :created` or `head :unprocessable_entity`)